meshapp/src/home/otp.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import {
+  Button,
+  Divider,
+  Grid,
+  Theme,
+  Typography,
+} from "@mui/material";
+import { ThemeProvider, createTheme } from "@mui/material";
+import { deepmerge } from "@mui/utils";
+import OtpInput from 'react-otp-input';
+import { axiosInstance } from "../config/axiosConfig";
+import { useCookies } from 'react-cookie';
+import { useEffect } from "react";
+import { errorHandler } from "../config/errorHandlerModule";
+import Swal from 'sweetalert2'
+
+const buttonTheme = createTheme({
+  components: {
+    MuiButton: {
+      styleOverrides: {
+        root: {
+          borderRadius: 25,
+          width: "100%",
+          height: "4.5rem",
+          fontSize: "1.5rem",
+          border: '2px solid #ccc',
+        },
+      },
+    },
+  },
+});
+
+const Otp = () => {
+  const [otp, setOtp] = useState(''); // inital OTP as empty
+  const navigate = useNavigate();
+  const [cookies, , removeCookie] = useCookies(['user_id']);
+
+  useEffect(() =>{},[cookies]);
```

Comment:
Similar to before, it would be nice if this line would have a comment.

Reply:
should be handled here 68b162a